index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;700&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/3167ce6dab.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="/css/normalize.css">
+    <link rel="stylesheet" href="/css/styles.css">
+</head>
+
+<body>
+    <header>
+        <div class="header container">
+            <h1 class="header-heading">My logo</h1>
+            <nav>
+                <i class="header-icon fa-solid fa-bars"></i>
+            </nav>
+        </div>
+    </header>
+    <section class="section-info">
```

Comment:
- [ ] Your first section is looking good, well done! However, the background should the one provided in the Figma template:

|Template|Your design|
|---|---|
|![Captura de pantalla 2022-04-21 a la(s) 7 13 43 p  m](https://user-images.githubusercontent.com/83717285/164590679-00b47bdc-5c5b-4e9e-bc99-4a43fe130c39.png) | ![Captura de pantalla 2022-04-21 a la(s) 10 19 32 p  m](https://user-images.githubusercontent.com/83717285/164590722-a9211a57-1b0e-46f6-ba77-fd9dc9ed41f6.png) |